server/src/app/lti-token-service.js

```diff
@@ -48,5 +48,32 @@ export const getCachedLTIToken = async (nonce, clientId, scope) => {
 };
 
 const cacheToken = async (token, nonce) => {
-  await insertNewAuthToken(nonce, `${token}`, 'lti_token');
+  if (nonce != null) {
```

Comment:
Would this be better just checking whether nonce is truthy? I don't see any other explicit equality checks for null in this code.
```suggestion
  if (nonce) {
```